plugin/pkg/admission/security/podsecuritypolicy/admission.go

```diff
@@ -95,10 +95,10 @@ func newPlugin(strategyFactory psp.StrategyFactory, failOnNoPolicies bool) *PodS
 	}
 }
 
-func (a *PodSecurityPolicyPlugin) SetExternalKubeInformerFactory(f informers.SharedInformerFactory) {
+func (plugin *PodSecurityPolicyPlugin) SetExternalKubeInformerFactory(f informers.SharedInformerFactory) {
```

Comment:
I was under impression that we only use single character names, but I agree that matching the rest of the functions is a good idea.